tests/add.bats

```diff
@@ -108,35 +108,62 @@ load helpers
 @test "add single file creates absolute path with correct permissions" {
   imgName=ubuntu-image
   createrandom ${TESTDIR}/distutils.cfg
+  permission=$(stat -c "%a" ${TESTDIR}/distutils.cfg)
 
   cid=$(buildah from --signature-policy ${TESTSDIR}/policy.json ubuntu)
   run_buildah add $cid ${TESTDIR}/distutils.cfg /usr/lib/python3.7/distutils
   run_buildah run $cid stat -c "%a" /usr/lib/python3.7/distutils
   root=$(buildah mount $cid)
-  expect_output "644"
+  expect_output $permission
   run_buildah commit --signature-policy ${TESTSDIR}/policy.json $cid containers-storage:${imgName}
   run_buildah rm $cid
 
   newcid=$(buildah from --signature-policy ${TESTSDIR}/policy.json ${imgName})
   run_buildah run $newcid stat -c "%a" /usr/lib/python3.7/distutils
-  expect_output "644"
+  expect_output $permission
   run_buildah rm $newcid
 }
 
 @test "add single file creates relative path with correct permissions" {
   imgName=ubuntu-image
   createrandom ${TESTDIR}/distutils.cfg
+  permission=$(stat -c "%a" ${TESTDIR}/distutils.cfg)
 
   cid=$(buildah from --signature-policy ${TESTSDIR}/policy.json ubuntu)
   run_buildah add $cid ${TESTDIR}/distutils.cfg lib/custom
   run_buildah run $cid stat -c "%a" lib/custom
   root=$(buildah mount $cid)
-  expect_output "644"
+  expect_output $permission
   run_buildah commit --signature-policy ${TESTSDIR}/policy.json $cid containers-storage:${imgName}
   run_buildah rm $cid
 
   newcid=$(buildah from --signature-policy ${TESTSDIR}/policy.json ${imgName})
   run_buildah run $newcid stat -c "%a" lib/custom
-  expect_output "644"
+  expect_output $permission
   run_buildah rm $newcid
 }
+
+@test "add with chown" {
```

Comment:
Is this really necessary as a separate test? Each separate test adds many seconds to the full run, and more if it has to pull an image. Why not just incorporate the `--chown` into the previous test? (line 131 above), and change the `stat` and `expect` in lines 132-134 to:
```bash
    run_buildah run $cid stat -c "%U:%G:%a" lib/custom
    expect_output "bin:bin:$permission" "ownership/mode of file copied into container"
```
(side note: the `buildah mount` in line 133 seems to be completely unnecessary. I think it's a copy-paste remnant. Feel free to remove it)

Reply:
Hi @edsantiago 
I checked that case but it is not a test for --chown flag but a basic tests for check buildah add file permissions. I'd like to keep per test cases to test a single function, so QE can find the problem quickly, and another reason I think we should keep a test case simple is the test case will be interrupted if one of the step failed and following steps will be cut off from the test results.

Reply:
I don't think I communicated my point well enough: I was trying to say that this test is almost identical to the previous one, and it would make sense to combine the two. Better readability and maintainability. But I won't block because of that.